app/views/posts/show.html.erb

```diff
@@ -1,5 +1,9 @@
 <p id="notice"><%= notice %></p>
 
+<h1>
+  <%= link_to @user.name, @user %>:
+</h1>
```

Comment:
Now that posts have user id's, getting the User info is WAY easier than it was going to be.